caluma-demo/app/styles/app.scss

```diff
@@ -9,4 +9,14 @@ $base-heading-font-weight: 300;
 $modal-z-index: 1;
 @import "ember-uikit";
 @import "ember-caluma";
-@import "ember-power-select";
\ No newline at end of file
+@import "ember-power-select";
+
+.cardtext {
+    overflow-wrap: break-word;
+    word-wrap: break-word;
+    hyphens: auto;
+}
```

Comment:
There are UIkit classes for this: https://getuikit.com/docs/text#text-wrapping

Reply:
I have removed these lines and added UIkit classes to the templates.